web/libs/editor/src/regions/Result.js

```diff
@@ -32,7 +33,7 @@ const Result = types
     // object tag
     to_name: types.late(() => types.reference(types.union(...Registry.objectTypes()))),
     // @todo some general type, maybe just a `string`
-    type: types.enumeration([
+    type: ff.isActive(ff.FF_CUSTOM_TAGS) ? types.late(() => types.enumeration([
```

Comment:
It might be easier to use an approach like this for such a large list:
```
(ff.isActive(ff.FF_CUSTOM_TAGS) ? type => types.late(() => type) : type => type)(types.enumeration([...]))
```
At least it's better to avoid some how duplication of the list itself. A new `const` is suitable as well.